### PR TITLE
Repo map

### LINKED ISSUE #59: Map out the repos
We need to list all the repos with a short description of what each one is for. I'm going to start this out as a standalone markdown file and link to it from the readme.

### PR DESCRIPTION
**Pull request by dylanPowers**
Resolves #59 

Still some things I'd like to tweak, but I'm open for opinions and suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,12 @@ IPFS could become a new major subsystem of the internet. If built right, it coul
 
 Please note that IPFS is a work in progress. It is an ambitious plan to make the internet more free, open, secure, and high performance. It builds on the good ideas of numerous battle-tested distributed systems.
 
-IPFS Talks:
+
+## [Project directory](project-directory.md)
+Lost? Searching for something? Need to submit an issue? Want to contribute? Take a peak at [project-directory.md](project-directory.md) which maps out all the repositories for the ipfs project on Github.
```
Comment (whyrusleeping) [inline README.md]: s/peak/peek/

File: project-directory.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+IPFS Project Directory
+======================
+Listed here are all the repositories for use by the ipfs community on Github.
```
Comment (whyrusleeping) [inline project-directory.md]: something about this phrasing is a tad strange

File: project-directory.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    
+* [website](https://github.com/ipfs/website)  
+    Source for the ipfs website at http://ipfs.io.
+    
+* [webui](https://github.com/ipfs/webui)  
+    The ipfs webui accessible from the api gateway typically at port 5001.
+
+
+## Demos and Examples
+* [container-demos](https://github.com/ipfs/astralboot)
```
Comment (whyrusleeping) [inline project-directory.md]: wrong link here.

File: project-directory.md (hunk 2 above)
Comment (dylanPowers) [inline project-directory.md]: Agreed

File: project-directory.md (hunk 3 above)
Comment (dylanPowers) [inline project-directory.md]: I need a new keyboard lol

**Comment by jbenet** [comment]
let's do this order

```
## Documents
* specs
* papers

##  Implementations
* go-ipfs
* node-ipfs

## Applications
* webui
* astralboot

## Demos and Examples
* examples
* container-demos

## community resources
* community
* infrastructure
* website
* mgmt
* logo
```

**Comment by jbenet** [comment]
LGTM other than order o/

**Comment by dylanPowers** [comment]
I've made my last changes. If everyone thinks it looks good, it can be merged.

**Comment by jbenet** [comment]
thanks @dylanPowers !

**Pull request merged**